src/plugins/discover/public/hooks/use_navigation_props.tsx

```diff
@@ -96,10 +96,10 @@ export const useNavigationProps = ({
 
   useEffect(() => {
     const dataViewId = typeof index === 'object' ? index.id : index;
-    services.locator
-      .getUrl({ dataViewId, ...buildParams() })
+    services?.locator
```

Comment:
ℹ️ Same here, this has to do with surrounding documents which we don't offer atm